src/crawler/scraper.py

```diff
@@ -193,7 +189,7 @@ def __save_results(self, score):
 
     def count_reviews(self):
         if self.count_reviews_monthly == "yes" or self.count_reviews_weekly == "yes":
-            obj = Count(self.name_of_app, self.package_name, self.country,
+            obj = Analytics(self.name_of_app, self.package_name, self.country,
```

Comment:
rename obj to analytics for better readblity

Reply:
okay